models/index.js

```diff
@@ -0,0 +1,10 @@
+var Sequelize = require('sequelize');
+
+var sequelize = new Sequelize(process.env.DB_URL)
+
+var models = [
+    'Todo',
+]
+models.forEach(function(model) {
+    module.exports[model] = sequelize.import(__dirname + '/' + model);
+});
```

Comment:
I removed index.js completely since I can just import the todo model directly. I hope this is what you had in mind